R/plate_plot.R

```diff
@@ -126,13 +128,18 @@ plate_plot <- function(data,
                        legend_n_row,
                        label_size,
                        silent = TRUE,
-                       scale) {
+                       scale,
+                       display_plot = TRUE) {
   if (!plate_size %in% c(6, 12, 24, 48, 96, 384)) {
     stop("Selected plate_size not available!")
   }
 
   if (missing(scale)) {
-    scale <- min((graphics::par("fin")[1] / 5.572917), (graphics::par("fin")[2] / 3.177083))
+    if (display_plot) {
+      scale <- min((graphics::par("fin")[1] / 5.572917), (graphics::par("fin")[2] / 3.177083))
+    } else {
+      scale <- 7 / 5.572917
```

Comment:
Why do you specifically use 7 here? I guess this should be dependent on the size of the plot that you want to save or not?

Reply:
Honestly, not sure why I added this change. I'm going to revert it

Reply:
Looking at my original PR message it seems I used this logic to suppress the `graphics::par()` function opening a new graphics device when I was actually just interested in the ggplot object (through `return(plot)`). The obvious and simpler solution to this problem is to simply pass a number (e.g. `7 / 5.572917 ~= 1.25`) as the `scale` argument to `plate_plot()`.

Long stroy short, this change should not be in the PR and reverting it in 855153f was the correct decision.